dxfeed/__init__.py

```diff
@@ -1,2 +1,3 @@
+from dxfeed.wrappers.DXFeed import DXFeed
```

Comment:
Please leave here only classes and methods from `wrappers` subpackage that would be available for the end-user